e2e/smoke.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from "@playwright/test";
+
+const baseURL = process.env.URL || "http://localhost:3000";
+test.beforeEach(async ({ page }) => {
+  await page.goto(baseURL);
+});
+
+test("Homepage has content and search works", async ({ page }) => {
+  await expect(page).toHaveTitle(/OwnPath/);
+  const search = page.locator("button:visible", {
+    hasText: "Search",
+  });
+
+  await page.locator("#desktop_zip").fill("80012");
```

Comment:
i think it'd be more robust to do `page.locator("input:visible")` here, so that it doesn't fail if we change playwright browser deets. and then for the check that results exist, have an id on the div wrapping both desktop + mobile results, or checking for pins on the map?

Reply:
ah i meant just having tests that aren't specific to one or the other

Reply:
i feel ok having this be desktop specific and just running it in one browser with desktop driver, since the point is to lightweight confirm site is up VS more in depth confirm functionality is retained.